### PR TITLE
align canary testing and pull request presubmit testing

### PR DESCRIPTION
**Pull request by morvencao**
- run presuite test in PR presubmit testing

### CODE REVIEW

**Comment by morvencao** [comment]
/do-not-merge

I reproduce the issue of observability-thanos-receive-default rollingUpdate, 
when we update the MCO CR, all instances will be stuck in `Terminating` status, while we have deploy PDB will maxUnavailable of 1.

@clyang82

File: pkg/tests/observability_install_test.go [hunk 1]
```diff
@@ -112,6 +115,41 @@ func installMCO() {
 		utils.PrintAllMCOPodsStatus(testOptions)
 	}
 
+	By("Checking placementrule CR is created")
```
Comment (clyang82) [inline pkg/tests/observability_install_test.go:118]: Please add `TODO` to remove the placementrule patch once the placementrule is implemented in server foundation.

**Comment by clyang82** [comment]
> when we update the MCO CR, all instances will be stuck in `Terminating` status, no new instance can be created, Also I found we have PDB will maxUnavailable of 1 deployed in the cluster, so it's weird why all instance are terminating.

Maybe there has conflicted so that PDB does not take effect really. is it possible?

File: pkg/tests/observability_install_test.go (hunk 1 above)
Comment (morvencao) [reply pkg/tests/observability_install_test.go:118]: done.

**Comment by morvencao** [comment]
I found that the `terminationGracePeriodSeconds` for thanos-receive pod is `900s`, see: https://github.com/thanos-io/kube-thanos/blob/0e6301db402437e3f5028efe06752cd34f206cd2/jsonnet/kube-thanos/kube-thanos-receive.libsonnet#L149
which may cause old pod can't be removed in 15 minutes, and it turn new old can't be recreated.

**Comment by clyang82** [comment]
> I found that the `terminationGracePeriodSeconds` for thanos-receive pod is `900s`, see: https://github.com/thanos-io/kube-thanos/blob/0e6301db402437e3f5028efe06752cd34f206cd2/jsonnet/kube-thanos/kube-thanos-receive.libsonnet#L149
> which may cause old pod can't be removed in 15 minutes, and it turn new old can't be recreated.

Can we send a PR to fix it or just open an issue to ask why it is 900s by default?

**Comment by morvencao** [comment]
`test-e2e` succeed bu the `test-e2e-gather-aws-console` failed, it's the prow env issue.

**Comment by morvencao** [comment]
The original PR for extend shutdown grace period to 900 is https://github.com/thanos-io/kube-thanos/pull/118

**Comment by morvencao** [comment]
@clyang82 PTAL

**Pull request merged**